src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -871,6 +871,246 @@ function bp_core_admin_debug_information( $debug_info = array() ) {
+			),
+			'BP_MESSAGES_AUTOCOMPLETE_ALL' => array(
+				'label' => 'BP_MESSAGES_AUTOCOMPLETE_ALL',
+				'value' => defined( 'BP_MESSAGES_AUTOCOMPLETE_ALL' ) && BP_MESSAGES_AUTOCOMPLETE_ALL ? __( 'Enabled', 'buddypress' ) : __( 'Disabled', 'buddypress' ),
+			),
+			'BP_GROUPS_SLUG' => array(
+				'label' => 'BP_GROUPS_SLUG',
+				'value' => defined( 'BP_GROUPS_SLUG' ) ? BP_GROUPS_SLUG : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_GROUPS_SLUG' ) ? BP_GROUPS_SLUG : 'undefined',
+			),
+			'BP_DISABLE_AUTO_GROUP_JOIN' => array(
+				'label' => 'BP_DISABLE_AUTO_GROUP_JOIN',
+				'value' => defined( 'BP_DISABLE_AUTO_GROUP_JOIN' ) && BP_DISABLE_AUTO_GROUP_JOIN ? __( 'Disabled', 'buddypress' ) : __( 'Enabled', 'buddypress' ),
+				'debug' => defined( 'BP_DISABLE_AUTO_GROUP_JOIN' ) ? BP_DISABLE_AUTO_GROUP_JOIN : 'undefined',
+			),
+			'BP_GROUPS_DEFAULT_EXTENSION' => array(
+				'label' => 'BP_GROUPS_DEFAULT_EXTENSION',
+				'value' => defined( 'BP_GROUPS_DEFAULT_EXTENSION' ) ? BP_GROUPS_DEFAULT_EXTENSION : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_GROUPS_DEFAULT_EXTENSION' ) ? BP_GROUPS_DEFAULT_EXTENSION : 'undefined',
+			),
+			'BP_MEMBERS_SLUG' => array(
+				'label' => 'BP_MEMBERS_SLUG',
+				'value' => defined( 'BP_MEMBERS_SLUG' ) ? BP_MEMBERS_SLUG : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_MEMBERS_SLUG' ) ? BP_MEMBERS_SLUG : 'undefined',
+			),
+			'BP_SIGNUPS_SKIP_USER_CREATION' => array(
+				'label' => 'BP_SIGNUPS_SKIP_USER_CREATION',
+				'value' => defined( 'BP_SIGNUPS_SKIP_USER_CREATION' ) && BP_SIGNUPS_SKIP_USER_CREATION ? __( 'Enabled', 'buddypress' ) : __( 'Disabled', 'buddypress' ),
+				'debug' => defined( 'BP_SIGNUPS_SKIP_USER_CREATION' ) ? BP_SIGNUPS_SKIP_USER_CREATION : 'undefined',
+			),
+			'BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH' => array(
+				'label' => 'BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH',
+				'value' => defined( 'BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH' ) ? BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH' ) ? BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH : 'undefined',
+			),
+			'BP_NOTIFICATIONS_SLUG' => array(
+				'label' => 'BP_NOTIFICATIONS_SLUG',
+				'value' => defined( 'BP_NOTIFICATIONS_SLUG' ) ? BP_NOTIFICATIONS_SLUG : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_NOTIFICATIONS_SLUG' ) ? BP_NOTIFICATIONS_SLUG : 'undefined',
+			),
+			'BP_BLOGS_SLUG' => array(
+				'label' => 'BP_BLOGS_SLUG',
+				'value' => defined( 'BP_BLOGS_SLUG' ) ? BP_BLOGS_SLUG : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_BLOGS_SLUG' ) ? BP_BLOGS_SLUG : 'undefined',
+			),
+			'BP_FRIENDS_SLUG' => array(
+				'label' => 'BP_FRIENDS_SLUG',
+				'value' => defined( 'BP_FRIENDS_SLUG' ) ? BP_FRIENDS_SLUG : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_FRIENDS_SLUG' ) ? BP_FRIENDS_SLUG : 'undefined',
+			),
+			'BP_FRIENDS_DB_VERSION' => array(
```

Comment:
I need to look at this one!! I don't see why it exists 🤔

Reply:
I confirm the constant `BP_FRIENDS_DB_VERSION` is deprecated and shouldn't be used anymore.